models/repo.go

```diff
@@ -708,17 +709,17 @@ func (repo *Repository) IsOwnedBy(userID int64) bool {
 }
 
 func (repo *Repository) updateSize(e Engine) error {
-	repoInfoSize, err := git.GetRepoSize(repo.repoPath(e))
+	size, err := util.GetDirectorySize(repo.repoPath(e))
```

Comment:
I think we might need a migration for this.

Reply:
I don't think there should be a need to modify anything about the existing database or contents, the method which the size is calculated will just be this different version on next push.